packages/lesswrong/components/comments/CommentsItem/CommentsItemDate.tsx

```diff
@@ -60,15 +60,18 @@ const CommentsItemDate = ({comment, post, tag, classes, scrollOnClick, scrollInt
   const { location, query } = useLocation();
   const { captureEvent } = useTracking();
 
+  const url = commentGetPageUrlFromIds({postId: post?._id, postSlug: post?.slug, tagSlug: tag?.slug, commentId: comment._id, tagCommentType: comment.tagCommentType, permalink})
+
   const handleLinkClick = (event: React.MouseEvent) => {
```

Comment:
`handleLinkClick` is only called if `scrollOnClick` is true. It just appends the query string to the url of the current page so that the whole page doesn't reload. Previously this would fail if the link was actually supposed to take you to a different page